docs/guide/exe_verilator.rst

```diff
@@ -563,6 +563,10 @@ Summary:
 
 .. option:: -fno-assemble
 
+.. option:: -fno-auto-var-split
```

Comment:
```suggestion
.. option:: -fno-var-split
```

I'd argue every optimization is "auto" by default ;)

I understand this is in comparison to the pragmas, but think it's understandable.